### DISCUSSION TITLE
a property of type object

### DISCUSSION THREAD

**Discussion by HimikoMorena** [UPBGE/upbge#1661] [Ideas/Features for UPBGE 0.3+]
it would be nice to have a property of type object, to use the same programming in logic nodes without having to duplicate it in another scene.

**Comment by HolandonHdunto** [comment]
You can probably do this using the group instance.

**Comment by rpaladin** [comment]
> it would be nice to have a property of type object, to use the same programming in logic nodes without having to duplicate it in another scene.

Could you describe your request some more. Are you asking for a `class` type node for creating custom object classes e.g. Player, Friend, Enemy, etc.

We also have these nodes if you didn't already know.

![image](https://user-images.githubusercontent.com/69180012/155585323-cd105489-f18b-4eaf-8dc1-4b744b3d8a71.png)
